tests/test_models.py

```diff
@@ -0,0 +1,52 @@
+"""Tests for the Post model."""
+
+from datetime import datetime
+from unittest.mock import patch
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.utils import timezone
+from blog.models import Post
+
+User = get_user_model()
+
+
+class TestPostModel(TestCase):
+    """Represent tests for the Post model."""
```

Comment:
Don't use verbs to describe objects. Classes don't do anything short of existing.

Reply:
Ok, but I have also noticed others examples like: "A class used to represent an Animal" in Real Python.
"This is a class to perform basic mathematical operations", "A class to represent a rectangle" - another sources.
In PEP 257 I didn't find it. However, I just wrote "Tests for the Post model."

Reply:
I think those examples are very artificial and might be misused for comments in that article. There may be exceptions, but in general, you don't need to repeat information that's already visible to the reader a few pixels above or below. Besides, a docstring title is supposed to be short. And prose goes into paragraphs separated from that title.

Reply:
Marking this as resolved.